sway-lib-std/src/token.sw

```diff
@@ -8,24 +8,32 @@ use ::tx::*;
 use ::context::call_frames::contract_id;
 use ::identity::Identity;
 
-/// Mint `amount` coins of the current contract's `asset_id` and transfer them to `destination` by calling either force_transfer_to_contract() or transfer_to_output(), depending on the type of `Identity`.
-pub fn mint_to(amount: u64, recipient: Identity) {
+/// Mint `amount` coins of the current contract's `asset_id` and transfer them
+/// to `to` by calling either force_transfer_to_contract() or transfer_to_output
```

Comment:
The newlines also need a `///`

Reply:
Not sure what you mean by the newlines...

Reply:
Ahhh wait my bad, GitHub was rending things funny. Carry on.